src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/custom.py

```diff
@@ -68,34 +70,17 @@ def db_create(
         **kwargs):
 
     # Verify edition
-    if 'Edition' in kwargs and kwargs['Edition'].lower() == 'datawarehouse':
-        raise CLIError('SQL Data Warehouse can be created with the command `az sql dw create`.')
+    edition = kwargs.get('edition')  # kwags['edition'] throws KeyError if not in dictionary
```

Comment:
The key here is lower cased while the original key is capitalized. I'm wondering if it is intentional. Unless the `kwargs` is a special dictionary, it compares the key in case-sensitive fashion.